app/controllers/Relation.scala

```diff
@@ -51,7 +60,7 @@ final class Relation(env: Env, apiC: => Api) extends LilaController(env):
           env.msg.api.postPreset(me, msg) >> rateLimited(msg.name)
         else
           api.follow(me, user.id).recoverDefault >> negotiate(
-            renderActions(user.name, getBool("mini")),
+            renderActions(user.name, getBool("mini"), getBool("showText")),
```

Comment:
?? Why is `showText` read from the HTTP request query parameters?

Reply:
In my initial fix I had not realised that when toggling e.g. follow/unfollow the text captions would re-appear as logic at runtime would not have the info if presented for paginated results or single user. Looking at the approach done for mini layout of the relation UI, I copied that approach.

Perhaps I could also attempt fixing this via introducing some additional routes?
e.g (/api/rel/follow/${user}/partial)